.github/release.yml

```diff
@@ -0,0 +1,18 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - octocat
+  categories:
+    - title: Breaking Changes 🛠
+      labels:
+        - Semver-Major
+        - breaking-change
```

Comment:
I think we should choose either semver-major/minor and breaking-change/enhancement and standardize the project around that instead of supporting both